src/Features/LanguageServer/Protocol/Workspaces/LspWorkspaceManager.cs

```diff
@@ -407,15 +410,27 @@ private static async ValueTask<bool> AreChecksumsEqualAsync(Document document, S
 
     #endregion
 
+    internal string? GetLanguageForUri(Uri uri)
```

Comment:
this needs to doc semantics.  LIke is this returning a roslyn LanguageName, or something else?